doc/source/nfloat.rst

```diff
@@ -0,0 +1,270 @@
+.. _nfloat:
+
+**nfloat.h** -- packed floating-point numbers with n-word precision
+===============================================================================
+
+This module provides binary floating-point numbers in a flat representation
+with precision in small fixed multiples of the word size
+(64, 128, 192, 256, ... bits on a 64-bit machine). The exponent range is
+close to a full word.
+A number with `n`-limb precision is stored using `n+2` limbs as follows:
+
+    +---------------+
+    | exponent limb |
+    +---------------+
+    |   sign limb   |
+    +---------------+
+    |  mantissa[0]  |
+    +---------------+
+    |      ...      |
+    +---------------+
+    | mantissa[n-1] |
+    +---------------+
```

Comment:
Looking at `nfloat_one`, I understand it as they are normalized in the same manner as the arf-objects (apart from removing trailing zero limbs). Could this perhaps be stated here?

Reply:
Yes, good point, that ought to be documented explicitly.